musicname/ze_tesv_skyrim_v4fix.cfg

```
@@ -0,0 +1,20 @@
+"music"
+{
+    "ancient_stones2.mp3"                                                           "Skyrim OST - Ancient Stones"
+    "audiomachine_warlords.mp3"                                                     "Audiomachine - Warlords"
+    "lopp1.mp3"                                                                     "Two Steps From Hell - Dark Ages"
+    "lopp2.mp3"                                                                     "Two Steps From Hell - Dark Ages"
+    "mus_combat_01.mp3"                                                             "Skyrim OST - Tooth and Claw"
+    "mus_combat_02.mp3"                                                             "Skyrim OST - Steel on Steel"
+    "mus_combat_06.mp3"                                                             "Skyrim OST - Death or Sovngarde"
+    "mus_combat_boss_02.mp3"                                                        "Skyrim OST - One They Fear"
+    "mus_combat_boss_03.mp3"                                                        "Skyrim OST - Watch the Skies"
+    "mus_town_day_01_loud.mp3"                                                      "Skyrim OST - The City Gates"
+    "mus_town_day_02.mp3"                                                           "Skyrim OST - From Past to Present"
+    "skyrim_ost silence_unbroken2.mp3"                                              "Skyrim OST - Silence Unbroken"
+    "two_staps_from_hell___protectors_of_the_world.mp3"                             "Two Steps From Hell - Protectors of the Earth"
+    "two_steps_from_hell___black_blade.mp3"                                         "Two Steps From Hell - Black Blade"
+    "two_steps_from_hell___the_last_stand__archangel___epic_action.mp3"             "Two Steps From Hell - The Last Stand (Archangel - Epic Action)"
```

I'm not sure if `The Last Stand` should also include `(Archangel)` in its name. It seems like album information which might not be needed.
```suggestion
    "two_steps_from_hell___the_last_stand__archangel___epic_action.mp3"             "Two Steps From Hell - The Last Stand (Archangel)"
```